PL-Policy.md

```diff
@@ -20,4 +20,55 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md).
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+Using the most current FedRAMP SSP template, 18F developed a system security plan which includes the cloud.gov PaaS and encompasses the cloud.gov applications. The security plan is developed in accordance with NIST Special Publication 800-18 R1 Guide of Developing Federal Information System Security Plans, as well as FedRAMP guidance. The System Security Plan: 
+
+1.	Is consistent with 18F enterprise cloud.gov architecture; 
+2.	Explicitly defines the authorization boundary for cloud.gov PaaS; 
+3.	Describes the operational context of cloud.gov PaaS in terms of missions and business processes; 
+4.	Provides the security category and impact level of cloud.gov including supporting rationale; 
+5.	Describes the operational network for cloud.gov; 
+6.	Describes relationships with or connections to other information systems; 
+7.	Provides an overview of the security requirements for cloud.gov; and 
+8.	Describes the security controls in place or planned for meeting those requirements including a rationale for the tailoring and supplementation decisions. 
+
+18F distributes copies of the cloud.gov security plan and communicates subsequent changes to the plan to the cloud.gov System Owner, ISSO, ISSM, AO and other designated members within the 18F staff and agency.
+The Information Security Officer, in conjunction with key 18F management officials, reviews the cloud.gov SSP at least annually or whenever there is a significant change to the information system.
+The Information Security Officer updates the cloud.gov SSP to address changes to the platform and its network of operation or problems identified during plan implementation or security control assessments, and thereafter whenever a significant change occurs. 
+The Information Security Officer protects the security plan from unauthorized modification by maintaining it within a version-controlled documentation repository (https://github.com/18F/cg-compliance ) that can only be modified by designated members from 18F and the agency.
```

Comment:
This part makes me pause since this realistically isn't our current procedure. We should probably say here and in the SSP that we can work on the SSP either as a write-access-controlled Google Doc _or_ as a version-controlled git repository.